src/RomModule.h

```diff
@@ -50,9 +56,28 @@ class RomModule: public PeripheralDevice, public ChipPIO8255
 
 		void ReadFromRom();
 
-	private:
+	protected:
 		BYTE *RomPack;
 };
+//---------------------------------------------------------------------------
+class MegaModule: public RomModule

```

Comment:
If it's possible, please separate this class to own file+header.

Reply:
MegaModule.h/cpp or RomMegaModule.h/cpp ?

Reply:
Good point! To keep that pattern, let's go with `RomMegaModule`